### ISSUE TITLE
Question - Example alert box is not announced in working feature branch. Is this expected behavior?

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26274]
## Description
The alert box is not being announced to JAWS or NVDA on Chrome b/c of the `role="alert"` on the error messages. Is this a current standard behavior on the forms? Can we manage focus and set it on the `Alertbox`, or is that on teams to manage outside this component?

## Acceptance Criteria
- [ ] Discuss the question in staging review or offline
- [ ] Identify the use case and ticket improvement as needed

**Comment by humancompanion-usds** [comment]
@SarahKay8 - This one is over a year old and doesn't provide a lot of context. But given that you just updated the guidance here, and that this was prioritized by VFS Teams for us to work on, can you give this a quick look and see if perhaps we can close it or ask for more details from the author? Thanks!

**Comment by humancompanion-usds** [comment]
I filed https://github.com/department-of-veterans-affairs/va.gov-team/issues/55907 in order to get this one re-investigated.

**Comment by Andrew565** [comment]
Addressed by #55907, closing as obsolete.

**Comment by humancompanion-usds** [comment]
No. 55907 has the guidance from Sarah on what we need to fix.

**Comment by Andrew565** [comment]
Will carry over due to not having enough time to start and complete this sprint.

**Comment by caw310** [comment]
Will carry over as work on this just started on the last day of the sprint.

**Comment by powellkerry** [comment]
The alert is not announced with screen readers on page load. This is working as expected, as defined in the W3C spec: "`role=alert` ... is equivalent to using `aria-live=assertive`" which means that the alert will not be announced until the content of the element changes or the role is dynamically added to the element. The example from the supplemental form pages is an exception in that the alert is used to give immediate feedback to the user. For cases like this the responsible team should implement a way to dynamically change content or add the role attribute dynamically so that the alert is read on or soon after page load.

See:
https://www.w3.org/WAI/GL/wiki/Using_ARIA_role%3Dalert_or_Live_Regions_to_Identify_Errors
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role
